app/src/main/res/values-en/strings.xml

```diff
@@ -2,21 +2,21 @@
 <!-- This is the source file for the app's strings in en-US.
      All other strings.xml are imported from POEditor crowd translation platform.
 
-     The strings are sorted into categories so to
+     The strings are sorted into categories to
      - avoid merge conflicts (people tend to append strings at the end of the file)
-     - better overview over the strings, better context for translators (strings are presented in
+     - better organize the strings to make it easier for translators (strings are presented in
        POEditor in the same order as in this file)
      -->
 <resources xmlns:tools="http://schemas.android.com/tools">
 
     <!-- Store description -->
 
     <string name="store_listing_short_description" tools:ignore="UnusedResources">"OpenStreetMap surveyor app"</string>
-    <string name="store_listing_full_description" tools:ignore="UnusedResources">"Help to improve the OpenStreetMap with StreetComplete!
+    <string name="store_listing_full_description" tools:ignore="UnusedResources">"Help improve OpenStreetMap with StreetComplete!
 
-This app finds incomplete and extendable data in your vicinity and displays it on a map as markers. Each of those is solvable by answering a simple question to complete the info on site.
+This app finds missing map data in your vicinity and displays it on a map as quests. Each quest is solvable by visiting the location on-site and answering a simple question to update the map.
```

Comment:
While we’re adjusting this string, it would be an opportunity to avoid passive voice (a bigger issue in English than in some other languages perhaps):

```suggestion
This app finds missing map data in your vicinity and displays it on a map as quests. Solve each quest by visiting the location on-site and answering a simple question to update the map.
```